coalib/results/result_actions/IgnoreResultAction.py

```diff
@@ -0,0 +1,35 @@
+from coalib.results.result_actions.ShowPatchAction import ShowPatchAction
+from coalib.results.result_actions.ResultAction import ResultAction
+import os.path
+
+
+class IgnoreResultAction(ResultAction):
+
+    SUCCESS_MESSAGE = "Result ignored successfully."
+
+    is_applicable = staticmethod(ShowPatchAction.is_applicable)
+
+    def apply(self, result, original_file_dict, file_diff_dict):
+        """
+        Ignore result
+        """
+        comment_delimiter = ""
+        for filename in result.diffs:
+            file_extension = os.path.splitext(os.path.basename(filename))[1]
+            file_extension = file_extension[1:]
+            if file_extension == "py":
+                file_extension = "python3"
+            file_coalang = file_extension + ".coalang"
+            f_coalang = open("/usr/lib/python3.5/site-packages/" +
```

Comment:
This file doesn't always exist. For example coala in a virtualenv, coala with a diff version number, windows, Mac, etc. etc.
This exists only on your laptop...